pkg/ipamd/rpc_handler.go

```diff
@@ -320,6 +321,18 @@ func (s *server) DelNetwork(ctx context.Context, in *rpc.DelNetworkRequest) (*rp
 	return &rpc.DelNetworkReply{Success: err == nil, IPv4Addr: ipv4Addr, IPv6Addr: ipv6Addr, DeviceNumber: int32(deviceNumber)}, err
 }
 
+func (s *server) GetNetworkPolicyConfigs(ctx context.Context, e *emptypb.Empty) (*rpc.NetworkPolicyAgentConfigReply, error) {
```

Comment:
We should add a comment here that this `GetNetworkPolicyConfigs` is invoked from the Network Policy Agent.